test/common_node/job_finished_test.cpp

```diff
@@ -0,0 +1,160 @@
+#include <gtest/gtest.h>
+
+#include <chrono>
+#include <cinttypes>
+#include <nlohmann/json.hpp>
+
+#include "rclcpp/logging.hpp"
+#include "rclcpp/node.hpp"
+#include "rclcpp/executors.hpp"
+
+#include "common_package/common_node.hpp"
+
+#include "interfaces/msg/job_finished.hpp"
+
+using namespace common_lib;
+
+class JobFinishedCommonNode : public CommonNode
+{
+public:
+    JobFinishedCommonNode(const std::string &name) : CommonNode(name)
+    {
+    }
+
+    inline void activate()
+    {
+        CommonNode::activate();
+    }
+
+    inline void job_finished(const uint8_t error_code, const nlohmann::json payload)
+    {
+        CommonNode::job_finished(error_code, payload);
+    }
+
+    inline void job_finished(const std::string error_message)
+    {
+        CommonNode::job_finished(error_message);
+    }
+
+    inline void job_finished()
+    {
+        CommonNode::job_finished();
+    }
+};
+
+TEST(common_package, job_finished_successfull)
+{
+    rclcpp::executors::SingleThreadedExecutor executor;
+
+    std::shared_ptr<JobFinishedCommonNode> common_node = std::make_shared<JobFinishedCommonNode>("common_node");
+    common_node->activate();
```

Comment:
Hier vllt vorher noch ein ASSERT_FALSE
```suggestion
    ASSERT_FALSE(common_node->get_active());
    common_node->activate();
```

Reply:
Ist gemacht